CONTRIBUTING.md

```diff
@@ -1,26 +1,20 @@
 # Contributing
 
-Please take a moment to review this document before submitting a pull request.
+Please review this document before submitting a pull request.
 
 ## Coding standards
 
-Our code formatting rules are defined in .prettierrc. You can automatically format your code against these standards by running:
-
-```bash
-yarn format
-```
+Our code formatting rules are defined in .prettierrc. Use [Prettier](https://prettier.io/) to format your code against these standards by running `package.json`'s `format` script. Method varies by Node.js package manager, e.g. `yarn format`.
```

Comment:
I don't think we need to specify the "method varies by package manager" wording. The yarn/npm/pnpm commands are easily interchangeable.